x-pack/plugin/logsdb/src/main/java/org/elasticsearch/xpack/logsdb/LogsdbIndexModeSettingsProvider.java

```diff
@@ -28,7 +28,7 @@ final class LogsdbIndexModeSettingsProvider implements IndexSettingProvider {
         Setting.Property.NodeScope
     );
     private static final String LOGS_PATTERN = "logs-*-*";
-    private volatile boolean isLogsdbEnabled;
+    private boolean isLogsdbEnabled;
```

Comment:
I think this need field needs to stay volatile? It can be updated via cluster settings update at runtime.

Reply:
Yes sorry, unintended.